packages/indexer/src/handlers/events/substrateChains/handleResponseEvent.handler.ts

```diff
@@ -20,19 +25,124 @@ export async function handleSubstrateResponseEvent(event: SubstrateEvent): Promi
 		})}`,
 	)
 
-	const host = getHostStateMachine(chainId)
+	const sourceId = formatChain(source_chain.toString())
+	const destId = formatChain(dest_chain.toString())
+
+	logger.info(
+		`Chain Ids: ${JSON.stringify({
+			sourceId,
+			destId,
+		})}`,
+	)
+
+	if (!isSubstrateChain(sourceId)) {
+		logger.error(`Skipping hyperbridge aggregated response`)
+		return
+	}
```

Comment:
The rest of this code won't run for GetRequests, actually it think it makes more sense to run the indexer when we observe the[ `GetRequestHandled`](https://github.com/polytope-labs/hyperbridge/blob/main/modules/pallets/ismp/src/lib.rs#L489) event on hyperbridge.

Reply:
Since we already update the status in the `GetRequestHandled` event, I'd say the rest of the code is actually not needed. But this code incorrectly handles `PostResponses` which i think we can tackle in a different PR.